### PR TITLE
[Do Not Review] Norm schedule like tensor

### PR DESCRIPTION
**Pull request by csarofeen**
Playing with new scheduling concepts. For the fusion problem:
```.py
import torch.nn.functional as F
import torch.nn as nn
import torch
import os

os.environ['PYTORCH_NVFUSER_DISABLE_FALLBACK'] = '1'

torch._C._jit_set_nvfuser_enabled(True)
torch._C._jit_set_texpr_fuser_enabled(False)
torch._C._jit_set_profiling_executor(True)
torch._C._jit_set_profiling_mode(True)
torch._C._jit_override_can_fuse_on_cpu(False)
torch._C._jit_override_can_fuse_on_gpu(False)
torch._C._jit_set_bailout_depth(20)

class BertOutput(nn.Module):
    def __init__(self, hidden_size, intermediate_size, hidden_dropout_prob, layer_norm_eps=1e-05):
        super().__init__()
        self.dense = nn.Linear(intermediate_size, hidden_size)
        self.LayerNorm = nn.LayerNorm(hidden_size, eps=layer_norm_eps)
        self.dropout = nn.Dropout(hidden_dropout_prob)
    def forward(self, hidden_states, input_tensor):
        hidden_states = self.dense(hidden_states)
        hidden_states = self.dropout(hidden_states)
        hidden_states = self.LayerNorm(hidden_states + input_tensor)
        return hidden_states
model = BertOutput(1024, 4096, 0.1)
model.to('cuda')
jit_model = torch.jit.script(model)
inputs = torch.randn([128, 128, 4096], device='cuda', requires_grad=True)
resids = torch.randn([128, 128, 1024], device='cuda', requires_grad=True)
for idx in range(0, 2) :
    outputs = jit_model.forward(inputs, resids)

start = torch.cuda.Event(enable_timing=True)
end = torch.cuda.Event(enable_timing=True)
start.record()
for idx in range(0, 5) :
    outputs = jit_model.forward(inputs, resids)
end.record()
torch.cuda.synchronize()
print("Jit time: ", start.elapsed_time(end))

start.record()
for idx in range(0, 5) :
    outputs = model.forward(inputs, resids)
end.record()
torch.cuda.synchronize()
print("Eager time: ", start.elapsed_time(end))
```
Overall perf gain is marginal, however bias-dropout-layernorm fusion on Titan V goes from 2.25 ms (126GB/s eff bandwidth) to 504us (566 GB/s eff bandwidth)

### CODE REVIEW

**Comment by csarofeen** [comment]
Stale, changes like this were merged in and we're using these scheduling primitives particularly in the prototype branch: https://github.com/csarofeen/pytorch/pull/814 which is getting moved into devel
